scripts/set_vars.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+APP_ROOT_PATH=$(cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && cd .. && pwd)
+
+export APP_ROOT_PATH="${APP_ROOT_PATH}"
+export SC_WEB_PATH="${APP_ROOT_PATH}"
```

Comment:
This will fail brutally, it should be `${SC_WEB_PATH}` if I understand it correctly
```suggestion
export SC_WEB_PATH="${SC_WEB_PATH}"
```

Reply:
Why? It won't work

Reply:
This will fail when `APP_ROOT_PATH` is set to the custom project root (i.e. ostis-web-platform-based projects), because that wouldn't be the path to the sc-web folder